### ISSUE TITLE
Cypress support file (`cypress.js`) is included twice

### ISSUE THREAD

**Issue by nemanjaglumac** [metabase/metabase#22712]
**Describe the bug**
Somehow the output from https://github.com/metabase/metabase/blob/master/frontend/test/__support__/e2e/cypress.js is included twice in the tests.

**To Reproduce**
1. open `https://github.com/metabase/metabase/blob/master/frontend/test/__support__/e2e/cypress.js`
2. Anywhere in that file write `console.log("foo");`
3. Run any of the Cypress tests in the open mode and observe the console
4. "foo" is printed twice

**Expected behavior**
Output of the support file should be included only once.

**Additional context**
- Originally noticed this behavior while reviewing another PR - please see [this comment](https://github.com/metabase/metabase/pull/21727#issuecomment-1126227540)
- If I remove [`supportFile`](https://github.com/metabase/metabase/blob/master/frontend/test/__support__/e2e/cypress.json#L5) from Cypress config in `cypress.json` or set it to false, all tests still work fine and I can see only one console log from repro steps
    - This feels weird and cannot be the solution before we figure out WHY is this happening in the first place

**Comment by alxnddr** [comment]
@nemanjaglumac it is included twice because we import it in almost each test since it re-exports helpers. If we export them directly, or re-export through another file then it will be included only once by Cypress
